### ISSUE TITLE
\ef doesn't work

### ISSUE THREAD

**Issue by hlinnaka** [greenplum-db/gpdb#3076]
```
postgres=# create function foofunc() returns int4 as $$ select 123; $$ language sql;
CREATE FUNCTION
postgres=# \ef foofunc
ERROR:  function pg_catalog.pg_get_functiondef(integer) does not exist
```

**Comment by hlinnaka** [comment]
`\ef` requires the `pg_get_functiondef` backend function, but we won't get that until PostgreSQL 8.4. I don't think we have much choice but disable \ef in GPDB 5.

**Comment by hlinnaka** [comment]
Fixed by PR #3078.
